go.mod

```diff
@@ -10,20 +10,20 @@ require (
 	github.com/oklog/run v1.1.0
 	github.com/prometheus/client_golang v1.15.1
 	github.com/prometheus/client_model v0.4.0
-	github.com/prometheus/common v0.43.0
+	github.com/prometheus/common v0.44.0
 	github.com/prometheus/exporter-toolkit v0.10.0
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.15.0
-	github.com/stretchr/testify v1.8.2
+	github.com/stretchr/testify v1.8.3
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	k8s.io/component-base v0.26.4
+	k8s.io/api v0.26.5
+	k8s.io/apimachinery v0.26.5
+	k8s.io/client-go v0.26.5
+	k8s.io/component-base v0.26.5
```

Comment:
Just to be clear, we are going to bump these (and other `k8s.io/*` deps) to `0.27.x` in the next major release (`v3`)? Also, is the contrary also true, i.e., to upgrade to the next minor `k8s.io/*` version, we need to create a major release?

Reply:
We will bump them in v2.10.0 to 0.27.x, for k8s deps we should raise minor early in the development cycle.

Reply:
Ah, so we're pointing to the latest patch `k8s` versions in this release, and only then are we going to point them to the next minor version (in the next-to-next release)?

Reply:
Yes, will bump it after v2.9 is released

Reply:
ACK, can you PTAL at [this](https://github.com/kubernetes/kube-state-metrics/pull/2076#pullrequestreview-1436982645) as well?